### ISSUE TITLE
Missing release for neo4j 4.0.7

### ISSUE THREAD

**Issue by gwvandesteeg** [neo4j-contrib/neo4j-apoc-procedures#4187]
## Guidelines

Please note that GitHub issues are only meant for bug reports/feature requests. If you have questions on how to use APOC, please ask on [the Neo4j Discussion Forum](https://community.neo4j.com/c/neo4j-graph-platform/procedures-apoc) instead of creating an issue here.

## Expected Behavior (Mandatory)

4.0.7 exists on https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json

## Actual Behavior (Mandatory)

4.0.7 does not exist on https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json

## How to Reproduce the Problem

Look at
https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json

Steps (Mandatory)
Go to https://neo4j-contrib.github.io/neo4j-apoc-procedures/versions.json
See Neo4j 4.0.7 missing from list
Know that Neo4j released 4.0.7

## Specifications (Mandatory)

Currently used versions 4.0.7, cannot upgrade to a newer version at this time.

**Comment by gwvandesteeg** [comment]
4.1 breaks our existing queries, various previously working queries now fail. Fully read only queries fail to execute due to being in a ReadTransaction.
4.2 breaks on our triggers created with `apoc.trigger.add`, they were working in 4.0, don't work with 4.2.

**Comment by Lojjs** [comment]
@gwvandesteeg Do you mean APOC 4.4.0.7? Due to a misstake, this release was broken and a new release 4.4.0.8 was released instead. So the reason this release is missing is that it should not be used. The JSON file you are referring to explains which APOC release you should be using depending on your Neo4j version. If you are on Neo4j 4.4.6, you should use APOC 4.4.0.6, while when you upgrade to Neo4j 4.4.7, you should use APOC 4.4.0.8. Does that help?

**Comment by vga91** [comment]
@gwvandesteeg  

Closed for lack of feedback, please feel free to reopen it in case you still have problems

**Comment by gwvandesteeg** [comment]
> @gwvandesteeg Do you mean APOC 4.4.0.7? Due to a misstake, this release was broken and a new release 4.4.0.8 was released instead. So the reason this release is missing is that it should not be used. The JSON file you are referring to explains which APOC release you should be using depending on your Neo4j version. If you are on Neo4j 4.4.6, you should use APOC 4.4.0.6, while when you upgrade to Neo4j 4.4.7, you should use APOC 4.4.0.8. Does that help?

No, it appears you completely misunderstood the provided information. As I stated, we are using Neo4J 4.0.7, the only version of Neo4J that actually works for us.
If you look at the linked version.js you will find apoc libraries for Neo4J 3.5.35, 4.1.0, 4.1.1, etc.. there are no apoc libraries present for the neo4j 4.0.x series of releases.

**Comment by vga91** [comment]
I don't know why a 4.0.x version was never released at the time, 
in any case unfortunately APOC currently supports only 4.4 (i.e. the LTS of 4.x) and 5.x versions, 
earlier ones are no longer supported,
consistent with the support of the corresponding Neo4j versions: https://neo4j.com/developer/kb/neo4j-supported-versions/
